.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
```

Comment:
Please capitalize commit messages. Kindly read [this](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) article and make sure cthat ommit messages are professional.

Reply:
capitalised commit messages to the best of my understanding